easy_my_coop/models/partner.py

```diff
@@ -179,5 +179,8 @@ def get_cooperator_from_email(self, email):
                                                ('email', '=', email)])
 
     def get_cooperator_from_crn(self, company_register_number):
+        if not company_register_number:
+            return None
+
         return self.env['res.partner'].search([('cooperator', '=', True),
                                                ('company_register_number', '=', company_register_number)])
```

Comment:
The logic is good but 

1. we prefer testing on positive expression where possible ("if <statement>" rather than "if not <statement>")
2. we prefer having closed `if ... else` statement
3. we cleaning up the code as we go, if you could make the files you touch pep 8 compliant (or even better, blacken the file in a separate commit), it'd be great.

That would give us:

```python
    def get_cooperator_from_crn(self, company_register_number):
        if company_register_number:
            partner = self.env["res.partner"].search(
                [
                    ("cooperator", "=", True),
                    ("company_register_number", "=", company_register_number),
                ]
            )
        else:
            partner = None
            
        return partner
```

Reply:
@enricostano I bring your suggestion taking into account the remark of @robinkeunen in this PR https://github.com/coopiteasy/vertical-cooperative/pull/98 

It will be merged this weeek I hope so you can close this one ;)

Reply:
@houssine78 many thanks! I couldn't find the time to finish this :(